src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2031,7 +2031,6 @@ internal enum ErrorCode
 
         #region diagnostics introduced for C# 11.0
 
-        // PROTOTYPE(raw-string-literals): renumber these if necessary.
         ERR_RawStringNotInDirectives = 9100,
```

Comment:
We can compact this all up. I'd simply go with 8996 since we're a day or two from merging